lesson-02-assignment.rb

```diff
@@ -0,0 +1,87 @@
+#Question 1
+CodingClass.find_each do |coding_class|
+   Course.create!(
+    coding_class_id: coding_class.id , 
+    trimester_id: '8', 
+    max_enrollment: '25')
+ end
+
+ #I accidentally created dublicate of Fall 26 trimester and delete it
+    Trimester.find_by(id: 9)&.destroy
+
+ #Question 2 
+    Student.create!(first_name: "Val", last_name: "Kot", email:"valkot@yahoo.com")
+
+ #Steps to find mentor with less than 2 enrollments
+    #1. Added to mentor model:
+        #has_many :mentor_enrollment_assignments
+        #has_many :enrollments, through: :mentor_enrollment_assignments 
+        Mentor.all.find{:enrollments.count <= 2 } 
```

Comment:
this isn't valid, you probably want to do:
```suggestion
  Mentor.all.find { |m| m.enrollments.count <= 2 }
```